release/servicing/nanoserver1809/meta.json

```diff
@@ -14,6 +14,9 @@
         },
         {
             "Tag": "2004"
+        },
+        {
+            "Tag": "20H2"
```

Comment:
Same here, this is servicing.  We should not have it here 100%, but maybe for LTS if @TravisEz13 agrees.

Reply:
Works for me, removed for servicing. Pending review for LTS.